master/cmd/determined-master/init.go

```diff
@@ -116,6 +116,9 @@ func registerConfig() {
 	registerString(flags, name("root"),
 		defaults.Root, "static file root directory")
 
+	registerBool(flags, name("development"),
```

Comment:
blocking: I think we should just add something like:
```
// IsDev returns if the current version is a dev version.
func IsDev() bool {
	return strings.Contains(Version, "dev")
}
```
to `master/version/version.go`. We don't want anyone to use this in a release version. Thoughts?

Reply:
yeah sounds good I didn't go that route because I wasn't sure if that'd always be available. Would there be a case when we want to develop against a release version master? especially as the WebUI don't need to rely on the potentially unstable latest master.

I guess it should be safe to have such an option especially with the default being of it being off. :thinking: let's slack :)

Reply:
> I guess it should be safe to have such an option especially with the default being of it being off.

As we talked about, I'm ok with this being a config (it doesn't matter _that_ much to me either way). One suggestion, though, add it to the config but don't add this flag so we can use it via the `master.yaml` but it doesn't show up in `determined-master --help`.

Reply:
Also, I'm in favor of config being `cors`, to avoid this getting used elsewhere (and then so it _just_ affects CORS, not every "development" config).

Reply:
I changed it to `--enable-cors <bool>`  which also handles the config and env variables. I'll look into seeing if there is a clean way to exclude it from showing up as a CLI option

Reply:
If you want a flag, then it'd be too big of a pain probably. But if it's ok to just have it configurable via the `master.yaml`, you should be able to just drop this line.

Reply:
Either way, `--enable-cors` seems good.

Reply:
:+1:  I took it out the cli option portion: https://github.com/determined-ai/determined/pull/913/commits/cfe5fa4bc8b54be49a7911bce154ea61924a9356